lib/ui/tabs/bookmark_tab.dart

```diff
@@ -10,41 +10,48 @@ class BookmarkTab extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     final GlobalBloc globalBloc = GlobalProvider.of(context);
-    return PagewiseListView(
-      padding: EdgeInsets.all(8.0),
-      pageSize: batchSize,
-      pageFuture: (pageIndex) {
-        return Future<List<Map<String, dynamic>>>(() async {
-          print('db fetch...');
-          //fetch
-          List<Map<String, dynamic>> documents =
-              await globalBloc.sqlite.getSavedEvents(batchSize, pageIndex);
-
-          return documents;
-        });
-      },
-      noItemsFoundBuilder: (context) {
-        return Column(
-          children: <Widget>[
-            Text(
-              "No Bookmarks",
-              style: TextStyle(
-                fontSize: 26.0,
-              ),
-            ),
-            Divider(),
-            Text("\nClick on"),
-            Icon(
-              Icons.bookmark_border,
-              color: Colors.yellow[800],
+    return kIsWeb
+        ? Center(
+            child: Text(
+              "This feature not available on web",
```

Comment:
Please do the following changes:
- Update the message to say: `Bookmarks are not available on web.`
- Add the bookmark icon above the message (increase the size of icon)

Also please share a screenshot so that I can review this.

Reply:
Added it @mannprerak2 
Screenshot:
![Screenshot from 2020-10-01 01-15-12](https://user-images.githubusercontent.com/45841802/94732204-93c9b100-0383-11eb-9e6b-41ca5940382d.png)

Reply:
Please do this as well
- Add the bookmark icon above the message (increase the size of icon) Or You can add anything you think will look good as the message alone looks empty.

And then share another SS after completing this.

Reply:
Yes sure. Will do it by tommorow morning 🌞. Good night 💤

Reply:
Changes has been done!
Screenshot:
![Screenshot (41)](https://user-images.githubusercontent.com/45841802/94772577-9229d880-03d7-11eb-8bcf-2d540b6a2654.png)

Reply:
Can you also increase the Icon size?